scripts-dev/check_newsfragment.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+#
+# A script which checks that an appropriate news file has been added on this
+# branch.
+
+echo -e "+++ \033[32mChecking newsfragment\033[m"
+
+set -e
+
+pr="$1"
```

Comment:
if you're expecting an argument, please could you add a comment to explain the script's usage, and (preferably) make it fail with a sensible error if the argument is not given.

Reply:
you're right, it's a bit sloppy and I guess just because it's an auxiliary script doesn't mean it has to be sloppy :)